src/main/java/thestonedturtle/lootlogger/ui/UniqueItemPanel.java

```diff
@@ -48,15 +49,16 @@
 @Getter
 class UniqueItemPanel extends JPanel
 {
-	private final float alphaMissing = 0.35f;
+	private float alphaMissing;
```

Comment:
This can be converted to a local variable, this and `alphaHas` should really have been static final.

Reply:
Is this how you mean?